samples/client.c

```diff
@@ -1213,18 +1213,19 @@ int main(int argc, char *argv[])
      * unmap the first group of the DMA regions
      */
     {
-        struct vfio_user_dma_region r[server_max_fds];
-        memcpy(r, dma_regions, sizeof(r));
-        for (i = 0; i < (int)ARRAY_SIZE(r); i++) {
-            r[i].flags = 0;
-            ret = tran_sock_msg(sock, 7, VFIO_USER_DMA_UNMAP, &r[i],
-                                sizeof(struct vfio_user_dma_region),
-                                NULL, NULL, 0);
+        for (i = 0; i < server_max_fds; i++) {
```

Comment:
should be nr_dma_regions

Reply:
This is from the time we could unmap multiple regions in a single request, `server_max_fds == nr_dma_regions / 2`.